kafka_topic_enforcer/src/main/java/com/tesla/data/topic/enforcer/Main.java

```diff
@@ -44,11 +52,4 @@ public static void main(String[] args) {
     final String cmd = commander.getParsedCommand();
     System.exit(commands.get(cmd).run());
   }
-
-  @Override
```

Comment:
this was a bug in prior commit, CLI doesn't initialize as expected unless we move the EnforceCommand extension to a dedicated class.